file-proxy/simple-file-proxy.js

```diff
@@ -0,0 +1,75 @@
+"use latest"
```

Comment:
I have made some minor changes to this file, perhaps it's easier to view these changes directly [here](https://github.com/yusinto/functions/blob/master/file-proxy/simple-file-proxy.js)